tests/test_pb.py

```diff
@@ -142,22 +142,186 @@ def test_PbFetcher_get_raises_if_get_throws_HTTPError_not_404(
             pb_fetcher.get_values(dummy_pv, jan_2018, jan_2018)
 
 
-def test_PbFileFetcher_get_pb_file_handles_pv_with_one_colon():
+def test_PbFileFetcher_get_all_pb_files_of_pv():
+    dummy_files = [
+        "root/LTS/a/b/c/d_2001.pb",
+        "root/MTS/a/b/c/d_2001_02_03.pb",
+        "root/STS/a/b/c/d_2001_02_03_04.pb",
+    ]
+
+    def side_effect(path_pv_dir):
```

Comment:
If you defined this function at the module level (ie move it out of `test_PbFileFetcher_get_all_pb_files_of_pv(`), then you would not need to repeat it later.